corehq/apps/analytics/static/analytix/js/hubspot.js

```diff
@@ -171,50 +171,24 @@ hqDefine('analytix/js/hubspot', [
                 });
             },
             onFormSubmit: function ($form) {
-                $('#get-trial-cta-calendar-content').fadeIn();
-                $('#get-trial-cta-form-content').addClass('hide');
-
                 var email = $form.find('[name="email"]').val(),
                     firstname = $form.find('[name="firstname"]').val(),
                     lastname = $form.find('[name="lastname"]').val(),
                     newUrl = document.location.href + '?email=' + email + '&name=' + firstname + '%20' + lastname;
 
                 kissmetrics.track.event("Get Trial Workflow - Contact Info Received");
-
+                
```

Comment:
small thing: looks like you by mistake added some whitespace here. We generally try to avoid whitespace only changes in a PR, and also generally stick to the standard that there should be no trailing whitespace at the end of a line and so blank lines shouldn't have any whitespace at all (other than the newline itself).